chia/wallet/wallet_state_manager.py

```diff
@@ -139,6 +141,18 @@ async def create(
             "pragma synchronous={}".format(db_synchronous_on(self.config.get("db_sync", "auto"), db_path))
         )
 
+        if self.config.get("log_sqlite_cmds", False):
+            sql_log_path = path_from_root(self.root_path, "log/wallet_sql.log")
+            self.log.info(f"logging SQL commands to {sql_log_path}")
+
+            def sql_trace_callback(req: str):
+                timestamp = datetime.now().strftime("%H:%M:%S.%f")
+                log = open(sql_log_path, "a")
+                log.write(timestamp + " " + req + "\n")
+                log.close()
```

Comment:
I recognize this is copy-pasted from what I did for the wallet. However, today I would have written this as:

```
with open(sql_lot_path, "a") as log:
   log.write(timestamp + " " + req + "\n")
```

Reply:
Yes, copy-paste is my well known superpower ;)
I'll change my PR according to your suggestion.